docs/getting_started.rst

```diff
@@ -0,0 +1,358 @@
+.. don't forget to set permissions hyperlink
+  + commands links + guide links
+
+.. _getting-started:
+
+===============
+Getting started
+===============
+
+If you recently installed Red, you should read this.
+This is a quick start guide for a general usage.
+
+.. note::
+
+    If you haven't installed Red, please do it by following
+    the :ref:`installation guides <main>`.
+
+Assuming you correctly installed Red, you should have a
+window like this:
+
+.. image:: .resources/red-console.png
+
+.. _gettings-started-invite:
+
+-------------------------
+Invite Red to your server
+-------------------------
+
+When started, the console will show you ``Invite URL`` (here at
+the bottom of the screenshot).
+Paste the link into your browser and select the server you want
+to invite the bot in, like any other bot.
+
+.. note:: You need the ``Manage server`` permission to add bots.
+
+Complete the captcha, it should tell you ``Authorized!`` and you
+should see your bot in the members list.
+
+.. attention::
+    If Discord shows ``Bot requires code grant``, please untick this
+    option in your token settings
+
+    .. image:: .resources/code-grant.png
+
+.. _getting-started-interact:
+
+-----------------
+Interact with Red
+-----------------
+
+As a chatbot, you interact with Red via the Discord text channels
+(not from the command prompt). For that, you will use the prefix you
+set before. For example, if your prefix is ``!``, you will execute your
+command like this: ``!ping``.
```

Comment:
The sentence starting with "For that" should probably be reworded to something like "To send commands to the bot, use the prefix you set before followed by the command you want to use.